samples/bluetooth/central_otc/src/main.c

```diff
@@ -237,13 +237,13 @@ static bool eir_found(struct bt_data *data, void *user_data)
 
 		for (i = 0; i < data->data_len; i += sizeof(uint16_t)) {
 			struct bt_le_conn_param *param;
-			struct bt_uuid *uuid;
+			struct bt_uuid *ots_uuid;
 			uint16_t u16;
 			int err;
 
 			(void)memcpy(&u16, &data->data[i], sizeof(u16));
-			uuid = BT_UUID_DECLARE_16(sys_le16_to_cpu(u16));
-			if (bt_uuid_cmp(uuid, BT_UUID_OTS) != 0) {
+			ots_uuid = BT_UUID_DECLARE_16(sys_le16_to_cpu(u16));
+			if (bt_uuid_cmp(ots_uuid, BT_UUID_OTS) != 0) {
```

Comment:
Same here. I'd look at the possibility of changing the global variable name that this local one shadows.